bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/DatetimePickerIconConfig.java

```diff
@@ -15,15 +15,15 @@ public class DatetimePickerIconConfig extends AbstractConfig {
 
     private static final long serialVersionUID = 1L;
 
-    private static final IKey<String> Up = newKey("up", FontAwesome5IconType.arrow_up_s.cssClassName());
-    private static final IKey<String> Down = newKey("down", FontAwesome5IconType.arrow_down_s.cssClassName());
-    private static final IKey<String> Date = newKey("date", FontAwesome5IconType.calendar_alt_r.cssClassName());
-    private static final IKey<String> Time = newKey("time", FontAwesome5IconType.clock_r.cssClassName());
-    private static final IKey<String> Previous = newKey("previous", FontAwesome5IconType.arrow_left_s.cssClassName());
-    private static final IKey<String> Next = newKey("next", FontAwesome5IconType.arrow_right_s.cssClassName());
-    private static final IKey<String> Today = newKey("today", FontAwesome5IconType.calendar_check_r.cssClassName());
-    private static final IKey<String> Clear = newKey("clear", FontAwesome5IconType.eraser_s.cssClassName());
-    private static final IKey<String> Close = newKey("close", FontAwesome5IconType.times_s.cssClassName());
+    private static final IKey<String> Up = newKey("up", FontAwesome6IconType.arrow_up_s.cssClassName());
+    private static final IKey<String> Down = newKey("down", FontAwesome6IconType.arrow_down_s.cssClassName());
+    private static final IKey<String> Date = newKey("date", FontAwesome6IconType.calendar_days_r.cssClassName());
+    private static final IKey<String> Time = newKey("time", FontAwesome6IconType.clock_r.cssClassName());
+    private static final IKey<String> Previous = newKey("previous", FontAwesome6IconType.arrow_left_s.cssClassName());
+    private static final IKey<String> Next = newKey("next", FontAwesome6IconType.arrow_right_s.cssClassName());
+    private static final IKey<String> Today = newKey("today", FontAwesome6IconType.calendar_check_r.cssClassName());
+    private static final IKey<String> Clear = newKey("clear", FontAwesome6IconType.eraser_s.cssClassName());
+    private static final IKey<String> Close = newKey("close", FontAwesome6IconType.xmark_s.cssClassName());
```

Comment:
If we want to support more than one version of FA then we need more complex type in `IBootstrapSettings` than the one I suggested earlier ...
And here those are all `static` fields which makes it harder to even make use of the settings class ...

Reply:
I wouldn't say it's solved, but at least worked around.